conf/upcloud.yml

```diff
@@ -0,0 +1,4 @@
+# Creates firewalls for all machines
+- hosts: localhost
+  roles:
+    - { role: upcloud-firewall, tags: ['upcloud','firewall'] }
```

Comment:
Linting fails here on Travis CI:

  4:50      error    too few spaces after comma  (commas)
  4:63      error    no new line character at the end of file  (new-line-at-end-of-file)